spec/design.md

```diff
@@ -42,3 +42,12 @@ The following principles guide and inform the design of the VT6 protocols and th
 4. **Platform independence** - The VT6 protocol tries to not rely on operating system specifics as much as possible (but consider the platform requirements noted in [`vt6/core0`](./draft/core0.md)).
 
 5. **Ease of use** - VT6 commands should be human-readable, and the protocol should be as easy to emit and consume in all programming and scripting languages as long as the ease of use does not significantly impede performance or resource requirements.
+
+## 3. Basic workflow example
+
+This example describes the example of a usual workflow when using a VT6-capable terminal.
+This terminal acts as a VT6 server.
+When it starts up it typically calls a VT6 shell which acts as a VT6 client.
+From this shell several more VT6 clients and legacy clients might be started.
+All clients talk to the original VT6 server, in this case the terminal.
+In order for legacy clients to be able to communicate with the VT6 server they are each wrapped by a VT6-capable wrapper which acts as a VT6 client.
```

Comment:
Not necessarily. Something like `cat` or `grep` does not need to care about VT6 at all. They just run as normal, they just use the stdin/stdout supplied by the shell.